### ISSUE TITLE
shell does not seem to work for remote repositories

### ISSUE THREAD

**Issue by ddgenome** [atomist/rug-cli#174]
First reported by @justinedelson, the Rug CLI shell no longer seems to work on remote repositories:

```
$ rug shell atomist-rugs:rug-editors
...
Resolving dependencies for atomist-rugs:rug-editors (latest·zip) completed
Loading atomist-rugs:rug-editors (0.25.0·zip) completed
Initializing shell for atomist-rugs:rug-editors (0.25.0·zip) failed

com.atomist.project.archive.Rugs.reviewers()Lscala/collection/Seq;
```

**Comment by justinedelson** [comment]
This may or may not be related, but I see the same error when doing a `rug install` with Rug 0.26.0.

Reproducible with https://github.com/justinedelson/aem-rugs if you edit `.atomist/manifest.yml` to change the version range to `[0.26.0,1.0.0)` (I have set the version range to `[0.25.3,0.26.0)` to temporarily work around this issue).

**Comment by cdupuis** [comment]
This is a problem with removal of `ProjectReviewer` in latest Rug 0.26.0 and is - unfortunately - expected to break the current version of the CLI.

Until we released a new version of the CLI, please use `--required 0.25.3``` or change you `manifest.yml` to point to a supported version.

**Comment by cdupuis** [comment]
Fixed in 0.33.0.
